lighthouse-core/test/results/sample_v2.json

```diff
@@ -1864,46 +1955,16 @@
       "id": "lcp-lazy-loaded",
       "title": "Largest Contentful Paint image was not lazily loaded",
       "description": "Above-the-fold images that are lazily loaded render later in the page lifecycle, which can delay the largest contentful paint. [Learn more](https://web.dev/lcp-lazy-loading/).",
-      "score": 1,
-      "scoreDisplayMode": "binary",
-      "details": {
-        "type": "table",
-        "headings": [
-          {
-            "key": "node",
-            "itemType": "node",
-            "text": "Element"
-          }
-        ],
-        "items": [
-          {
-            "node": {
-              "type": "node",
-              "lhId": "page-1-IMG",
-              "path": "3,HTML,1,BODY,14,IMG",
-              "selector": "body > img",
-              "boundingRect": {
-                "top": 574,
-                "bottom": 654,
-                "left": 132,
-                "right": 252,
-                "width": 120,
-                "height": 80
-              },
-              "snippet": "<img src=\"lighthouse-480x318.jpg?iar2\" width=\"120\" height=\"80\">",
-              "nodeLabel": "img"
-            }
-          }
-        ]
-      }
+      "score": null,
+      "scoreDisplayMode": "notApplicable"
```

Comment:
problem?

Reply:
I'd say yes, perhaps the shifter could shrink back to 0 after it expands?

Reply:
hm,, i've done this and it is still n/a.